### PR TITLE
feat / add userstream to KuCoin

### PR DESCRIPTION
**Pull request by vic-en**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue: #1804
- Related Clubhouse Story:

**A description of the changes proposed in the pull request**:
This PR adds usersrtream websocket to KuCoin.

### CODE REVIEW

File: hummingbot/market/kucoin/kucoin_api_user_stream_data_source.py [hunk 1]
```diff
@@ -56,89 +54,58 @@ async def get_listen_key(self):
...
-        return websockets.connect(stream_url)
+        return websockets.connect(stream_url, ping_interval=40, ping_timeout=self.PING_TIMEOUT)
+
+    async def _inner_messages(self, ws: websockets.WebSocketClientProtocol) -> AsyncIterable[str]:
+        while True:
+            try:
+                msg: str = await ws.recv()
+                self._last_recv_time = time.time()
+                yield msg
+            except ConnectionClosed:
```
Comment (Nullably) [inline hummingbot/market/kucoin/kucoin_api_user_stream_data_source.py:84]: You don't need to except these 2 exceptions if you don't do anything with them apart from raising them again.

Comment (vic-en) [reply hummingbot/market/kucoin/kucoin_api_user_stream_data_source.py:84]: True. I'll remove them now.

Comment (vic-en) [reply hummingbot/market/kucoin/kucoin_api_user_stream_data_source.py]: Done.

**Comment by dennisocana** [comment]
@vic-en please see attached log file below. Looks like Hummingbot crashed after running it the whole day. It's on AWS t2.xlarge instance - the server is OK but Hummingbot became unresponsive and closed the connection while trying to attach to the screen session.

[logs_test_kucoin.log](https://github.com/CoinAlpha/hummingbot/files/4945788/logs_test_kucoin.log)

**Comment by vic-en** [comment]
@dennisocana 
Let me know when you've confirmed fix so I can remove debug messages.

**Pull request merged**